spec/requests/sessions/sign_out_spec.rb

```diff
@@ -6,7 +6,7 @@
   subject { delete destroy_user_session_path, headers: auth_headers }
 
   context 'when given correct headers' do
-    let!(:auth_headers) { create_auth_headers }
+    let!(:auth_headers) { user.create_new_auth_token }
```

Comment:
here you wouldn't need to create headers again. inside `subject`, it should be called the helper method directly.